src/shared/codelens/localLambdaRunner.ts

```diff
@@ -423,43 +382,55 @@ export async function makeInputTemplate(params: {
     return inputTemplatePath
 }
 
-export async function executeSamBuild(params: {
-    baseBuildDir: string,
+export interface ExecuteSamBuildArguments {
+    baseBuildDir: string
     channelLogger: Pick<ChannelLogger, 'info'>,
     codeDir: string,
     inputTemplatePath: string,
     manifestPath?: string,
+    environmentVariables?: NodeJS.ProcessEnv,
     samProcessInvoker: SamCliProcessInvoker,
-}): Promise<string> {
-    params.channelLogger.info(
+}
+
+export async function executeSamBuild({
+    baseBuildDir,
+    channelLogger,
+    codeDir,
+    inputTemplatePath,
+    manifestPath,
+    environmentVariables,
+    samProcessInvoker
+}: ExecuteSamBuildArguments): Promise<string> {
+    channelLogger.info(
         'AWS.output.building.sam.application',
         'Building SAM Application...'
     )
 
-    const samBuildOutputFolder = path.join(params.baseBuildDir, 'output')
+    const samBuildOutputFolder = path.join(baseBuildDir, 'output')
 
     const samCliArgs: SamCliBuildInvocationArguments = {
         buildDir: samBuildOutputFolder,
-        baseDir: params.codeDir,
-        templatePath: params.inputTemplatePath,
-        invoker: params.samProcessInvoker,
-        manifestPath: params.manifestPath
+        baseDir: codeDir,
+        templatePath: inputTemplatePath,
+        invoker: samProcessInvoker,
+        manifestPath
     }
     await new SamCliBuildInvocation(samCliArgs).execute()
 
-    params.channelLogger.info(
+    channelLogger.info(
         'AWS.output.building.sam.application.complete',
         'Build complete.'
     )
 
     return path.join(samBuildOutputFolder, 'template.yaml')
 }
 
-export const invokeLambdaFunction = async (params: {
+export interface InvokeLambdaFunctionArguments {
```

Comment:
This property set is too large. What do you think about grouping the debug properties into an interface (because you're adding two properties in this change)?
isDebug
debugConfig
debugPort
debuggerPath